src/main/java/am/ik/yavi/core/ValueValidator.java

```diff
@@ -157,4 +161,14 @@ default X validated(T t, Locale locale, ConstraintGroup constraintGroup)
 		return (value, locale, constraintGroup) -> Validated.traverseOptional(value,
 				v -> validator.validate(v, locale, constraintGroup));
 	}
+
+	/**
+	 * @since 0.11.0
+	 */
+	static <A1, R> Validatable<A1> validatable(
```

Comment:
It causes a cyclic dependency between `ValueValidator` <-> `Validatable`.

`Validatable.from(ValueValidator)` should be better

Reply:
Refactored Validator.toValidatable as Validatable.from(ValueValidator) to fix cyclic dependency between ValueValidator and Validatable.